src/CONST.ts

```diff
@@ -821,6 +821,7 @@ const CONST = {
         PRIVATE_DOMAIN_ONBOARDING: 'privateDomainOnboarding',
         IS_TRAVEL_VERIFIED: 'isTravelVerified',
         NEWDOT_MULTI_FILES_DRAG_AND_DROP: 'newDotMultiFilesDragAndDrop',
+        PLAID_COMPANY_CARDS: 'plaidCompanyCards',
```

Comment:
FYI, adding this beta back end side in [this PR](https://github.com/Expensify/Web-Expensify/pull/46735).